src/load/azext_load/data_plane/utils/argtypes.py

```diff
@@ -0,0 +1,335 @@
+    type=str,
+    help="Name of the app component.",
+)
+
+app_component_type = CLIArgumentType(
+    validator=validators.validate_app_component_type,
+    options_list=["--app-component-type"],
+    type=str,
+    help="Type of resource of the app component.",
+)
+
+app_component_kind = CLIArgumentType(
+    options_list=["--app-component-kind"],
+    type=str,
+    help="Kind of the app component.",
+)
+
+server_metric_id = CLIArgumentType(
+    validator=validators.validate_metric_id,
+    options_list=["--metric-id"],
+    type=str,
+    help="Fully qualified ID of the server metric.",
+)
+
+server_metric_name = CLIArgumentType(
+    options_list=["--metric-name"],
+    type=str,
+    help="Name of the server metric.",
+)
+
+server_metric_namespace = CLIArgumentType(
+    options_list=["--metric-namespace"],
+    type=str,
+    help="Namespace of the server metric.",
+)
+
+server_metric_aggregation = CLIArgumentType(
+    options_list=["--aggregation"],
+    type=str,
+    help="Aggregation of the server metric.",
+)
+
+metric_name = CLIArgumentType(
+    options_list=["--metric-name"],
+    type=str,
+    help="Name of the metric.",
+)
+
+metric_namespace = CLIArgumentType(
+    validator=validators.validate_metric_namespaces,
+    completer=get_generic_completion_list(
+        utils.get_enum_values(models.AllowedMetricNamespaces)
+    ),
+    options_list=["--metric-namespace"],
+    required=True,
+    type=str,
+    help=f"Namespace of the metric. Allowed values: {', '.join(utils.get_enum_values(models.AllowedMetricNamespaces))}",
+)
+
+metric_dimension = CLIArgumentType(
+    options_list=["--metric-dimension"],
+    type=str,
+    help="Value of the metric dimension.",
+)
+
+start_iso_time = CLIArgumentType(
+    validator=validators.validate_start_iso_time,
+    options_list=["--start-time"],
+    type=str,
+    help="ISO 8601 formatted start time.",
+)
+
+end_iso_time = CLIArgumentType(
+    validator=validators.validate_end_iso_time,
+    options_list=["--end-time"],
+    type=str,
+    help="ISO 8601 formatted end time.",
+)
+
+interval = CLIArgumentType(
+    validator=validators.validate_interval,
+    completer=get_generic_completion_list(
+        utils.get_enum_values(models.AllowedIntervals)
+    ),
+    options_list=["--interval"],
+    type=str,
+    help=f"ISO 8601 formatted interval. Allowed values: {', '.join(utils.get_enum_values(models.AllowedIntervals))}",
+)
+
+aggregation = CLIArgumentType(
+    options_list=["--aggregation"],
+    type=str,
+    help="Operation used to aggregate the metrics",
+)
+
+dimension_filters = CLIArgumentType(
+    validator=validators.validate_dimension_filters,
+    options_list=["--dimension-filters"],
+    nargs="*",
+    help=(
+        "space and comma-separated dimension filters: key1[=value1,value2] [key2[=value3] ...]. "
+        "* is supported as a wildcard for both key and value. "
+        "Example: `--dimension-filters key1=value1,key2=*`, `--dimension-filters *`"
```

Comment:
we use space to separate by default instead of ','

Reply:
resolved